robowflex_library/scripts/cob4_multi_target.cpp

```diff
@@ -0,0 +1,108 @@
+/* Author: Juan D. Hernandez */
+
+// Robowflex
+#include <robowflex_library/io/visualization.h>
+#include <robowflex_library/log.h>
+#include <robowflex_library/detail/cob4.h>
+#include <robowflex_library/builder.h>
+#include <robowflex_library/planning.h>
+#include <robowflex_library/robot.h>
+#include <robowflex_library/scene.h>
+#include <robowflex_library/trajectory.h>
+#include <robowflex_library/util.h>
+
+using namespace robowflex;
+
+/* \file cob4_multi_target.cpp
+ * A script that demonstrates the multi-target IK query functionality and planning with a cob4 robot. The
+ * robowflex_resouces package needs to be available, see https://github.com/KavrakiLab/robowflex_resources.
+ * You should run RViz and have a RobotState visualization display enabled set to look at
+ * /robowflex/robot_description, and robowflex/state. Also a MarkerArray should be added to visualize the
+ * target IK poses.
+ */
+
+static const std::string LEFT_ARM = "arm_left";
+static const std::string RIGHT_ARM = "arm_right";
+static const std::string BOTH_ARMS = "both_arms";
+static const std::string LEFT_EE = "arm_left_7_link";
+static const std::string RIGHT_EE = "arm_right_7_link";
+
+int main(int argc, char **argv)
+{
+    // Startup ROS
+    ROS ros(argc, argv);
+
+    // Create the default Care-O-Bot4 robot.
+    auto cob4 = std::make_shared<Cob4Robot>();
+    cob4->initialize();
+
+    // Load kinematics for the COB4 robot.
+    cob4->loadKinematics(BOTH_ARMS);
+
+    // Fold the arms.
+    cob4->setGroupState(RIGHT_ARM, {2.69, 1.70, -0.91, 1.50, -2.14, -2.35, 1.06});
+    cob4->setGroupState(LEFT_ARM, {-1.14, -1.50, 0.34, -1.50, 0.43, -1.56, -1.20});
```

Comment:
Can you just set them all at once with BOTH_ARMS?

Reply:
Done!